app/src/main/java/com/limelight/Game.java

```diff
@@ -2307,4 +2320,20 @@ public boolean onKey(View view, int keyCode, KeyEvent keyEvent) {
                 return false;
         }
     }
+
+    public void disconnect() {
+        super.onBackPressed();
+    }
+
+    @Override
+    public void onBackPressed() {
+        // Instead of "closing" the game activity open the game menu. The user has to select
+        // "Disconnect" within the game menu to actually disconnect from the remote host.
+        //
+        // Use the onBackPressed instead of the onKey function, since the onKey function
+        // also captures events while having the on-screen keyboard open.  Using onBackPressed
+        // ensures that Android properly handles the back key when needed and only open the game
+        // menu when the activity would be closed.
```

Comment:
I think that's probably happening because of `StreamView.onKeyPreIme()` which is catching the back press prior to the IME.